numpy/core/src/umath/ufunc_object.c

```diff
@@ -105,7 +105,17 @@ _error_handler(int method, PyObject *errobj, char *errtype, int retstatus, int *
     PyObject *pyfunc, *ret, *args;
     char *name = PyBytes_AS_STRING(PyTuple_GET_ITEM(errobj,0));
     char msg[100];
-    NPY_ALLOW_C_API_DEF;
+
+    NPY_ALLOW_C_API_DEF
+
+    // don't need C API for a simple print
```

Comment:
I am sorry, I seem to have forgotten to mention that this also should use `/* blah */` style comments, `//` are C++ style comments.
After that, please squash the commits, and I think we can merge it (might want to post a message, otherwise nobody notices the update), if you can think of a way to add a test, that would be great, but I am not sure there is anything not overly complex.